### ISSUE TITLE
can't run chron-feed example 

### ISSUE THREAD

**Issue by sanchitram1** [farcasterxyz/hub-monorepo#906]
**What is the bug?**
Can't connect to a hub when running the chron-feed example, from the docs.

**How can it be reproduced? (optional)**
StackBlitz errored for me as well.  The first error I encountered was when I:
```sh
$ git clone https://github.com/farcasterxyz/hub-monorepo.git
$ cd packages/hub-nodejs/examples/chron-feed
$ npm install
$ npm start
```

**Additional context (optional)**
The first error I got was because of a missing `HUB_URL`.
```sh
Error: Could not parse target name ""
    at InternalChannel (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/internal-channel.js:113:19)
    at ChannelImplementation (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/channel.js:35:32)
    at Client (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/client.js:65:36)
    at ServiceClientImpl (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/make-client.js:58:5)
    at getSSLClient (file:///Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@farcaster/hub-nodejs/dist/index.mjs:6367:10)
    at getSSLHubRpcClient (file:///Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@farcaster/hub-nodejs/dist/index.mjs:6349:26)
    at <anonymous> (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/index.ts:105:18)
    at <anonymous> (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/index.ts:146:1)
    at ModuleJob.run (node:internal/modules/esm/module_job:192:25)
```

So I edited `index.ts` and hardcoded testnet1.farcaster.xyz:2281 as the URL (got from #882), and got this error.

```sh
sanch@Sanchits-MBP chron-feed % npm start

> example-chron-feed@0.0.0 start
> tsx index.ts

hub_url: testnet1.farcaster.xyz:2281
HubError: Connection failed: please check that the hub’s address, ports and authentication config are correct. No connection established
    at fromServiceError (file:///Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@farcaster/hub-nodejs/dist/index.mjs:6311:10)
    at Object.callback (file:///Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@farcaster/hub-nodejs/dist/index.mjs:6328:43)
    at Object.onReceiveStatus (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/client.js:192:36)
    at Object.onReceiveStatus (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:360:141)
    at Object.onReceiveStatus (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:323:181)
    at <anonymous> (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/resolving-call.js:94:78)
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
  errCode: undefined,
  presentable: false,
  [cause]: undefined
}
```

**TLDR**
What are all the running Hubs and their URLs?  And how do I find my farcaster fid to use as an input?

**Comment by varunsrin** [comment]
The first error is due to a missing Hub Url. I added one of our mainnet hubs in here to make the examples easier to run: https://github.com/farcasterxyz/hub-monorepo/pull/905

The second error is because the example uses SSL authentication while testnet hubs are not authenticated. Changing the auth methods should resolve the issue.

**Comment by varunsrin** [comment]
stackblitz is still failing, opened a ticket to track this: https://github.com/stackblitz/webcontainer-core/issues/1045

**Comment by sanchitram1** [comment]
First error is fixed, i'm able to get the chronological feeds when I use nemes.farcaster.xyz:2283

Second error -- I switched to [101 from 102](https://github.com/farcasterxyz/hub-monorepo/blob/b225a844104873749d654c7629fe043cda1e51f6/packages/hub-nodejs/examples/chron-feed/index.ts#L101-L102), and am using testnet1.farcaster.xyz:2283 and got this:
```sh
(node:89481) ExperimentalWarning: Custom ESM Loaders is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

node:internal/event_target:1036
  process.nextTick(() => { throw err; });
                           ^
Error: EAGAIN
    at new Worker (node:internal/worker:288:19)
    at new InternalWorker (node:internal/worker:464:5)
    at new HooksProxy (node:internal/modules/esm/hooks:496:20)
    at new CustomizedModuleLoader (node:internal/modules/esm/loader:351:18)
    at createModuleLoader (node:internal/modules/esm/loader:421:14)
    at get esmLoader [as esmLoader] (node:internal/process/esm_loader:15:26)
    at node:internal/modules/cjs/loader:119:47
    at node:internal/util:764:15
    at Module.load (node:internal/modules/cjs/loader:1128:26)
    at Module._load (node:internal/modules/cjs/loader:965:12)
Emitted 'error' event on InternalWorker instance at:
    at [kOnErrorMessage] (node:internal/worker:323:10)
    at [kOnMessage] (node:internal/worker:334:37)
    at MessagePort.<anonymous> (node:internal/worker:229:57)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:761:20)
    at exports.emitMessage (node:internal/per_context/messageport:23:28) {
  code: 'ERR_WORKER_INIT_FAILED'
}
```

**Comment by varunsrin** [comment]
testnet1.farcaster.xyz:2283 has SSL so you'll need to use the SSL client instead of the unauthenticated one. 

However, this error is not what I'd expect to see. Can you try it with the SSL client and see if it returns the same error?

**Comment by sanchitram1** [comment]
Hub not found.  Is the url correct?

```sh
> example-chron-feed@0.0.0 start
> tsx index.ts

hub_url: testnet1.farcaster.xyz:2283
HubError: NotFound: 
    at fromServiceError (file:///Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@farcaster/hub-nodejs/dist/index.mjs:6311:10)
    at Object.callback (file:///Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@farcaster/hub-nodejs/dist/index.mjs:6328:43)
    at Object.onReceiveStatus (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/client.js:192:36)
    at Object.onReceiveStatus (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:360:141)
    at Object.onReceiveStatus (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:323:181)
    at <anonymous> (/Users/sanch/workspace/hub-monorepo/packages/hub-nodejs/examples/chron-feed/node_modules/@grpc/grpc-js/build/src/resolving-call.js:94:78)
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
  errCode: 'not_found',
  presentable: false,
  [cause]: undefined
}
```

**Comment by sanchitram1** [comment]
Message in this case referring to the casts from the FID that I queried?  Ran it with 1, 2, 3 and got a NotFound.

**Comment by varunsrin** [comment]
yes, those fids haven't posted any messages to testnet

**Comment by sanchitram1** [comment]
Ok, do you know an example fid that has a message on TestNet?  I think my original issue is resolved.

**Comment by varunsrin** [comment]
there are a few accounts that broadcast messages on testnet. if you use the subscribe api's to listen, you should be able to see their fids

**Comment by sds** [comment]
Did you resolve your issue, @sanchitram1?

**Comment by sanchitram1** [comment]
Yes, you can close it!
